src/bindings/html/head.js

```diff
@@ -12,13 +12,17 @@ export function getMeta(head) {
   let appVersion = null;
 
   // XXX take last found instead of first?
-  const els = head.querySelectorAll(
+  const metas = head.querySelectorAll(
     'meta[name="availableLanguages"],' +
     'meta[name="defaultLanguage"],' +
     'meta[name="appVersion"]');
-  for (let el of els) {
-    const name = el.getAttribute('name');
-    const content = el.getAttribute('content').trim();
+
+  // Polyfill NodeList.prototype[Symbol.iterator] for Chrome.
+  // See https://code.google.com/p/chromium/issues/detail?id=401699
+  metas[Symbol.iterator] = Array.prototype[Symbol.iterator];
```

Comment:
It is, yes.  I'm not sure where to put it.  We need it in html/head and we might start using `for..of` in html/dom (but we don't right now).  Maybe add a new module, html/polyfill, which can be required by any other code?